COLLECTION.md

```diff
@@ -6,6 +6,7 @@
 
 |  #  | Brand              | Type        | Name                        | Origin      | Flavour                     |
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------------------|
+| 146 | -                  | [Green]     | [Anji White Tea]            | [China]     | Green tea                   |
```

Comment:
The wiki said it's green,  the chinese wiki I looked up, it is said it's usually green and only picked at the time it whitened, and that's why it's unique and rare.

http://baike.baidu.com/item/%E5%AE%89%E5%90%89%E7%99%BD%E8%8C%B6

80~85 degree for brewing.